plugins/inlinetoolbar/plugin.js

```diff
@@ -151,36 +184,43 @@
 			};
 
 			/**
-			 * Places the inline toolbar next to a specified element so the tip of the toolbar's triangle
-			 * points to that element.
-			 *
+			 * @inheritdoc CKEDITOR.ui.balloonPanel#attach
 			 * @member CKEDITOR.ui.inlineToolbarView
-			 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
 			 */
-			CKEDITOR.ui.inlineToolbarView.prototype.create = function( element ) {
-				this.attach( element, false );
-
-				var that = this,
-					editable = this.editor.editable();
-				this._detachListeners();
-
-				this._listeners.push( this.editor.on( 'resize', function() {
-					that.attach( element, false );
-				} ) );
-				this._listeners.push( editable.attachListener( editable.getDocument(), 'scroll', function() {
-					that.attach( element, false );
-				} ) );
+			CKEDITOR.ui.inlineToolbarView.prototype.attach = function( element, options ) {
+				this._element = element;
+				CKEDITOR.ui.balloonPanel.prototype.attach.call( this, element, options );
 			};
 
 			/**
 			 * Displays the inline toolbar, pointing it to the `element`.
 			 *
 			 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
+			 * @param {Boolean} [showToolbar=true] Show inline toolbar after attach.
 			 * @member CKEDITOR.ui.inlineToolbar
 			 */
-			CKEDITOR.ui.inlineToolbar.prototype.create = function( element ) {
+			CKEDITOR.ui.inlineToolbar.prototype.attach = function( element, showToolbar ) {
 				this._view.renderItems( this._items );
-				this._view.create( element );
+				this._view.attach( element, {
+					focusElement: false,
+					show: showToolbar === false ? false : true
```

Comment:
I see what you wanted to do here: to set false only if it was explicitly set, however it would be nice to support other _falsy_ values, like `0`.

How about changing `showToolbar` to `hidden`? This value would be marked as `false` by default, and then you could just put `!hidden` as `options.show`. WDYT?